aten/src/ATen/core/CustomTensorImpl.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <string>
+#include <unordered_map>
+#include <ATen/core/stack.h>
+#include <ATen/core/TensorBody.h>
+#include <c10/core/TensorImpl.h>
+
+using Impl = void(const c10::FunctionSchema& schema, torch::jit::Stack*);
+
+namespace at {
+
+class CAFFE2_API CustomTensorImpl : public TensorImpl {
```

Comment:
Besides the tagging, could you explain the other differences between this and OpaqueTensorImpl?

Reply:
I guess not much besides how it is meant to be used.